include/hipipe/core/python/utility/pyboost_cv_point_converter.hpp

```diff
@@ -0,0 +1,43 @@
+/****************************************************************************
+ *  hipipe library
+ *  Copyright (c) 2018, Iterait a.s.
+ *  Author(s) Adam Blazek
+ *
+ *  This file is distributed under the MIT License.
+ *  See the accompanying file LICENSE.txt for the complete license agreement.
+ ****************************************************************************/
+
+#pragma once
+#if defined HIPIPE_BUILD_PYTHON && defined HIPIPE_BUILD_PYTHON_OPENCV
+
+// this header has to be included before the numpy header
+#include <hipipe/core/python/initialize.hpp>
+
+#include <Python.h>
+#include <boost/python.hpp>
+#include <numpy/ndarrayobject.h>
+#include <opencv2/core/core.hpp>
+
+#include <cstdio>
```

Comment:
Please remove this one, I believe we don't use it in neither header nor source. Plus this header is kind of unusual in C++ (it is C-style input/output).